clang/compiler-course/markin_i_conversions/markin_i_conversions.cpp

```diff
@@ -0,0 +1,103 @@
+#include "clang/AST/ASTConsumer.h"
+#include "clang/AST/RecursiveASTVisitor.h"
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Frontend/FrontendPluginRegistry.h"
+#include "llvm/Support/raw_ostream.h"
+#include <map>
+#include <string>
+#include <vector>
+
+using namespace clang;
+using namespace std;
```

Comment:
Please, avoid using broad scope `using` directives. If you need to use `using`, consider adding this for particular functions, classes, etc.